zone/perl_client.cpp

```diff
@@ -3300,6 +3305,7 @@ void perl_register_client()
 	package.add("CanHaveSkill", &Perl_Client_CanHaveSkill);
 	package.add("CashReward", &Perl_Client_CashReward);
 	package.add("ChangeLastName", &Perl_Client_ChangeLastName);
+	package.add("ChangePetName", &Perl_Client_ChangePetName);
```

Comment:
This should be consistent with the underlying function in what it does `GrantPetNameChange`

```
package.add("GrantPetNameChange", &Perl_Client_GrantPetNameChange);
```